pyproject.toml

```diff
@@ -154,3 +154,4 @@ reportUnboundVariable = "warning"
 
 [tool.pytest.ini_options]
 filterwarnings = ["ignore::DeprecationWarning:gymnasium.*:"]
+addopts = "--ignore=gymnasium/wrappers/jax_to_numpy.py --ignore=gymnasium/wrappers/jax_to_torch.py"
```

Comment:
Does this ignore the tests as well?

Reply:
Ohh, I missed that. Maybe that is fine but we should note that somewhere

Reply:
The alternative is [this](https://github.com/Farama-Foundation/Gymnasium/blob/v1.0.0/gymnasium/wrappers/vector/record_episode_statistics.py#L26) but I'm not so sure that's a good idea since it seems possible to lose track of how many skips are in the repo itself.